packages/material-ui-unstyled/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -140,14 +140,18 @@ const Identity = (x) => x;
 // Safari, on iOS, supports touch action since v13.
 // Over 80% of the iOS phones are compatible
 // in August 2020.
+// Utilizing the CSS.supports method to check if touch-action is supported.
+// Since CSS.supports is supported on all but Edge@12 and IE and touch-action
+// is supported on both Edge@12 and IE if CSS.supports is not available that means that
+// touch-action will be supported
 let cachedSupportsTouchActionNone;
 function doesSupportTouchActionNone() {
   if (cachedSupportsTouchActionNone === undefined) {
-    const element = document.createElement('div');
-    element.style.touchAction = 'none';
-    document.body.appendChild(element);
-    cachedSupportsTouchActionNone = window.getComputedStyle(element).touchAction === 'none';
-    element.parentElement.removeChild(element);
+    if (CSS && CSS.supports) {
+      cachedSupportsTouchActionNone = CSS.supports('touch-action: none');
```

Comment:
I think this crashes in Edge considering

> Edge doesn't support parentheses-less one-argument version.
-- https://developer.mozilla.org/en-US/docs/Web/API/CSS/supports

```suggestion
      cachedSupportsTouchActionNone = CSS.supports('touch-action', 'none');
```

Though I honestly have trouble deciphering what "parentheses-less one-argument version" means.

Reply:
Ah ok, I was checking the usage and did it as suggesed. Thanks.